packages/algob/src/types.ts

```diff
@@ -310,6 +310,14 @@ export interface MultiSig {
   v: number
 }
 
+// represents Lsig object fetched directly from raw file
+export interface RawLsig {
+  l: Uint8Array
+  args?: Object // optional
+  sig?: Object | undefined // optional
```

Comment:
if you use `?` then you don't need to add `| undefined`

Reply:
Noted